packages/channel-provider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@statechannels/channel-provider",
-  "version": "0.0.3",
+  "version": "0.0.5",
```

Comment:
what happened to 0.0.4?

Reply:
@lalexgap published it but did not bump the code